psa-ext/patch.cddl

```diff
@@ -0,0 +1,13 @@
+comid.psa-fw-patch-triples = 5
+
+$$triples-map-extension //= (
+  comid.psa-fw-patch-triples => one-or-more<psa-fw-patch-triple-record>
+)
+
+security-critical = bool
+
+psa-fw-patch-triple-record = [
```

Comment:
Presumably fw-update triple is a TODO, right?

Reply:
I have removed this in 8a6a99e (i.e., blocklists)